### PR TITLE
Backend integration

### PR DESCRIPTION
**Pull request by romesh-jaya**

### CODE REVIEW

File: .env.dev
Comment (walterheymans) [inline .env.dev:29]: Is it possible to do comments here? Maybe just add a comment saying that this should be updated to point to the API server if possible.

**Review by walterheymans — APPROVED** [review]
Thanks for making the changes. This looks good to me now. I made one comment inline regarding a comment. But if it is not possible to add a comment there then its okay.

Before we merge this, I would also like to get Jacques' approval.

**Comment by walterheymans** [comment]
@JPBeukes please take another look at this when you get time. As mentioned previously, I think the reason it is not working for you is because you did not change the `BACKEND_ENDPOINT` environment variable. Please try this: `BACKEND_ENDPOINT=http://45.62.11.172`.

Chrome based browsers might also give you some issues, because it is HTTP. Please ask if you cannot get it to work, so that we can help.

P.S. The daily graph will be broken now. The reason is that we updated a field in the SQL database from `varchar` to `timestamp`, and now the Python script fails to insert new values. I will fix this next week.

File: .env.dev
Comment (romesh-jaya) [reply .env.dev:30]: Fixed

**Comment by romesh-jaya** [comment]
> @JPBeukes please take another look at this when you get time. As mentioned previously, I think the reason it is not working for you is because you did not change the `BACKEND_ENDPOINT` environment variable. Please try this: `BACKEND_ENDPOINT=http://45.62.11.172`.
> 
> Chrome based browsers might also give you some issues, because it is HTTP. Please ask if you cannot get it to work, so that we can help.
> 
> P.S. The daily graph will be broken now. The reason is that we updated a field in the SQL database from `varchar` to `timestamp`, and now the Python script fails to insert new values. I will fix this next week.

@walterheymans just a heads up that you can run the following DB query and then immediately after that, check the daily graph. That is, until you made the changes to the data insertion script: 

ALTER TABLE tswaps.TRADEDATA
  MODIFY UPDATED_TIME TIMESTAMP;

**Review by walterheymans — APPROVED** [review]

**Pull request closed without merging**
